src/subject/readMESubject(mostafa).md

```diff
@@ -0,0 +1 @@
+I complete subject class with setcode ,setnamea,getcode and get name
```

Comment:
this is not a valid ReadMe file 
readme files are for documentation [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes)